va_explorer/users/forms.py

```diff
@@ -231,8 +233,12 @@ def save(self, commit=True):
             # See allauth:
             # https://github.com/pennersr/django-allauth/blob/c19a212c6ee786af1bb8bc1b07eb2aa8e2bf531b/allauth/account/utils.py
             # setup_user_email(self.request, user, [])
-
-            get_adapter().send_new_user_mail(self.request, user, password)
+            if email_confirmation:
+                get_adapter().send_new_user_mail(self.request, user, password)
+                
+            else:
+                print(f"="*20)
+                print(f"Created user with email {user.email} and temp. password {password}")
```

Comment:
Should there be a comment here mentioning how this is a work-around for the current email server blocking certain outbound email to certain providers and we'd like to eventually not log temp passwords?

Reply:
yes, definitely. I will expose this argument to the management command so choolwe can automatically send out emails when running it